compiler/lcalc/to_ocaml.ml

```diff
@@ -396,18 +396,21 @@ let rec format_expr (ctx : decl_ctx) (fmt : Format.formatter) (e : 'm expr) :
         | Div_int_int | Div_rat_rat | Div_mon_mon | Div_mon_int | Div_mon_rat
         | Div_dur_dur ->
           Format.fprintf ppf "%a@ " format_pos (Expr.pos (List.nth args 1))
-        | HandleExceptions ->
-          let excs =
-            match args with [(EArray ex, _)] -> ex | _ -> assert false
-          in
-          Format.fprintf ppf "[|@[<hov>%a@]|]@ "
-            (Format.pp_print_list
-               ~pp_sep:(fun ppf () -> Format.fprintf ppf ";@ ")
-               format_pos)
-            (List.map Expr.pos excs)
+        (* | HandleExceptions ->
+         *   let excs =
+         *     match args with [(EArray ex, _)] -> ex | _ -> assert false
+         *   in
+         *   Format.fprintf ppf "[|@[<hov>%a@]|]@ "
+         *     (Format.pp_print_list
+         *        ~pp_sep:(fun ppf () -> Format.fprintf ppf ";@ ")
+         *        format_pos)
+         *     (List.map Expr.pos excs) *)
```

Comment:
Why is this commented? And below too?

Reply:
Ah, well spotted, this is a leftover and should be removed: the HandleExceptions operator used to need a specially crafted list of exceptions, but now that is all handled at runtime so no special treatment is required anymore.